docs/architecture/VERSIONING_AND_COMPATIBILITY.md

```
@@ -0,0 +1,578 @@
+| 变更类型 | 示例 | 处理方式 |
+|---------|-----|---------|
+| 删除字段 | 删除 `actionPath` | 经 deprecation → 新版本 |
+| 重命名字段 | `painId` → `signalId` | 经 deprecation alias → 新版本 |
+| 必选字段类型变更 | `string` → `number` | 新版本 schema |
+| 添加必选字段 | 加上没有 default 的 `Required` 字段 | 新版本 + 数据迁移 |
+| 收窄现有字段 | union 缩小 | 谨慎，需调研使用情况 |
+| 改变状态机转换 | 增加新状态 | 新版本 + 迁移规则 |
+
+### 3.3 Schema 版本递增的命名
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)